cnf-certification-test/identifiers/identifiers.go

```diff
@@ -250,6 +250,18 @@ The label value is not important, only its presence.`,
 		false,
 		TagExtended)
 
+	TestStorageRequiredPods = AddCatalogEntry(
+		"storage-required-pods",
```

Comment:
"storage-required-pods" seems weird to me for this test case. What about "no-pvs-on-localstorage".